packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -795,36 +821,50 @@ class ServiceExtensionManager {
       return;
     }
 
-    assert(value != null);
-    if (value is bool) {
-      // TODO(kenz): stop special casing http timeline logging once it can be
-      // called via `callServiceExtension`. See
-      // https://github.com/dart-lang/sdk/issues/43628.
-      if (name == extensions.httpEnableTimelineLogging.extension) {
-        await _service.forEachIsolate((isolate) async {
-          await _service.httpEnableTimelineLogging(isolate.id, value);
-        });
-      } else {
+    final callExtension = () async {
+      assert(value != null);
+      if (value is bool) {
+        // TODO(kenz): stop special casing http timeline logging once it can be
```

Comment:
High level question (see lines 831-833): should we be calling the httpTimeline dart.io extension on all isolates or just the selected isolate like we do for other service extensions? Or on the flip side, should we be calling the other service extensions on all isolates like we are currently doing for the http timeline extension? @bkonyi @jacob314

Reply:
i think it is reasonable to call this for all isolates. The other extensions are flutter specific and only apply to the main isolate.